test/cli/incremental-resolver/incremental-resolver.out

```diff
@@ -48,6 +48,14 @@ test/cli/incremental-resolver/expect-failures/constant_override.rb:111: Redefini
 test/cli/incremental-resolver/expect-failures/constant_override.rb:110: Method `e` does not exist on `T.class_of(<root>)` https://srb.help/7003
      110 |B = e
               ^
+  Got T.class_of(<root>) originating from:
+    test/cli/incremental-resolver/expect-failures/constant_override.rb:110:
+     110 |B = e
+     111 |module B
+     112 |  extend T::Sig
+     113 |  sig { returns(T.all(B,T)) }
+     114 |  def foo; T.unsafe(nil); end
+     115 |end
```

Comment:
Similarly, it looks like the loc we use for `<self>` when it's the file top level is a loc meaning "the entire file" (because there is implicitly a class def (with a declLoc) around the entire file contents' AST)